test/lit/passes/string-lowering-instructions.wast

```diff
@@ -24,6 +24,9 @@
 
   ;; CHECK:      (import "wasm:js-string" "fromCodePoint" (func $fromCodePoint_5 (type $5) (param i32) (result (ref extern))))
 
+  ;; CHECK:      (memory $m 1)
+  (memory $m 1)
```

Comment:
Why does this test need a memory? I don't seem to see any memory-using instructions.

Reply:
Hmm, you're right. Will remove.